lib/models/category.rb

```diff
@@ -3,5 +3,5 @@ class Category
 
   property :id, Serial, key: true
   property :name, String
-  has n, :dishes, :model => 'Dish', :child_key => [:categoryId]
+  has n, :dishes
```

Comment:
Why was the extra code there in the first place, is it no longer needed - just out of curiosity